lib/bolt/node/ssh.rb

```diff
@@ -171,7 +171,9 @@ def write_remote_file(source, destination)
     end
 
     def make_tempdir
-      result = execute('mktemp -d')
+      command = 'mktemp -d'
+      command = "TMPDIR=#{@tmpdir} #{command}" if @tmpdir
```

Comment:
This doesn't seem to actually change the resulting dir I get from `mktemp -d` on my mac. It works on EL 7.

Reply:
hmmm maybe I should just ditch mktemp when using a tmpdir

command = "mkdir #{@tmpdir}/#{random_dir}" ?